### DISCUSSION TITLE
the ebpf slack channel is not working

### DISCUSSION THREAD

**Discussion by sprappcom** [cilium/ebpf#1527] [Q&A] [answered]
https://cilium.herokuapp.com/

**Comment by lmb** [comment]
The correct link is now https://slack.cilium.io/ Where did you find that link?

**Reply by florianl** [reply, marked as answer]
There are some references in this repository to https://ebpf.io/slack, which redirects to https://cilium.herokuapp.com/.

Not sure who manages this DNS redirect, but I also opened an issue around this with https://github.com/ebpf-io/ebpf.io-website/issues/621.

**Reply by lmb** [reply]
I'll raise this internally.
